examples/vite.config.js

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
     alias: {
       // Resolve symlink ourselves
       '@react-three/fiber': resolve('node_modules', '@react-three', 'fiber'),
+      '@react-three/cannon': resolve('../'),
```

Comment:
I had to add this for it to correctly link to the local build

Reply:
```suggestion
      '@react-three/cannon': resolve('node_modules', '@react-three', 'cannon'),
```